src/main/java/org/opensearch/ad/transport/StatsAnomalyDetectorTransportAction.java

```diff
@@ -193,16 +198,17 @@ private Map<String, Object> getClusterStatsMap(ADStatsRequest adStatsRequest) {
      * Make async request to get the Anomaly Detection statistics from each node and, onResponse, set the
      * ADStatsNodesResponse field of ADStatsResponse
      *
-     * @param client Client
+     * @param sdkRestClient SDKRestClient
      * @param listener MultiResponsesDelegateActionListener to be used once both requests complete
      * @param adStatsRequest Request containing stats to be retrieved
      */
     private void getNodeStats(
-        Client client,
+        SDKRestClient sdkRestClient,
         MultiResponsesDelegateActionListener<ADStatsResponse> listener,
         ADStatsRequest adStatsRequest
     ) {
-        client.execute(ADStatsNodesAction.INSTANCE, adStatsRequest, ActionListener.wrap(adStatsResponse -> {
+        ADStatsNodeRequest adStatsNodeRequest = new ADStatsNodeRequest(adStatsRequest);
+        sdkRestClient.execute(ADStatsNodesAction.INSTANCE, adStatsNodeRequest, ActionListener.wrap(adStatsResponse -> {
```

Comment:
Isn't the request here should be `ADStatsRequest`? Why are we making a new one?

Reply:
ADStatsNodeRequest is created because:
1) The nodeOperationMethod in [ADStatsNodesTransportAction](https://github.com/joshpalis/anomaly-detection/blob/fd547014fdde5114bbc9c8e49fe7aaa37eb6e793/src/main/java/org/opensearch/ad/transport/ADStatsNodesTransportAction.java#L100) take ADStatsNodeRequest in as argument. 
2) We have called nodeOperationMethod from doExecute  method.  I don't wanted to make any change in existing methods in order to maintain the sanity of the work. So I created ADStatsNodeRequest in the above method and passed it in the transportAction call.